types/react-dom/experimental.d.ts

```diff
@@ -37,6 +37,7 @@ declare module "." {
 
 declare module "react" {
     interface ViewTransitionInstance {
+        getComputedStyle: () => CSSStyleDeclaration;
```

Comment:
This is wrong. This is on each `Animatable` below.

It's more like an interface that extends `Animatable`. Kind of like an Element, a PsuedoElement if you will.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/compare/34d56f354c32700fe16b073c9cc0b3dc8eefa141..dad3d859a49490ac8fbd67f60055ec0b6cbdd66b